### PR TITLE
ci: add arm docker images

### PR DESCRIPTION
**Pull request by martadams89**
Add arm64 images to ci pipeline

### CODE REVIEW

**Comment by acasajus** [comment]
- can you update the action definition so it's also triggered when the workflow file changes so we can make sure this change doesn't break anything?

**Comment by martadams89** [comment]
@acasajus thanks, I've updated to run only on

```
on:
  push:
    paths:
      - '.github/workflows/main.yml'
```

This means the workflow will only run when this file is updated - not sure if this is what you were asking for?

**Comment by martadams89** [comment]
Anything you need me to do on the PR so that the action can be tested?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ name: Test and lint
 
 on:
   push:
+    paths:
+      - '.github/workflows/main.yml'
```
Comment (acasajus) [inline .github/workflows/main.yml:6]:
```suggestion
on: [push, pull_request]
```

Comment (acasajus) [reply .github/workflows/main.yml:6]: Let's try this.

**Pull request merged**